snuba/query/validation/validators.py

```diff
@@ -255,3 +258,58 @@ def validate(self, query: Query, alias: Optional[str] = None) -> None:
             raise InvalidQueryException(
                 f"granularity must be multiple of {self.minimum}"
             )
+
+
+class TagConditionValidator(QueryValidator):
```

Comment:
This should have a test of its own rather than relying on the api integration test